modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -552,79 +552,31 @@ func (suite *KeeperTestSuite) TestChanUpgradeAck() {
 	}
 }
 
-func (suite *KeeperTestSuite) TestChanUpgradeOpen() {
```

Comment:
woah am I in merge-hell and not seeing things right or is this getting overwritten?

Reply:
Argh I'll check, maybe I was in merge hell